mantidproject/mantid
# Add Dx tab in the data window

**Description of work.**
When a workspace has Dx data, a tab is added in the data window so these Dx values can be seen.



**To test:**
- Create a workspace containing Dx data or run `ws = CreateWorkspace(DataX="1,2,3", DataY="4,5", Dx="6,7")`
- Open the data and check that the new tab is here.
- Open some other data and check no tab is present if it is not needed.


Fixes #30011 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
